pluto_build/bin/config.sh

```diff
@@ -31,34 +31,22 @@ function urlparse {
 }
 
 function FGDB_export {
+  name=$1
```

Comment:
Now that this export looks so similar to SHP_export there's probably an opportunity for DRYing this, don't know how worth putting the effort in is now

Reply:
The bash code we maintain is overall extremely not DRY. This is an issue we inherited from the Baiyue/Molly days, and my bash skills have never been good enough to refactor any of it.